packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -489,25 +490,26 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
             let networkTime = -1; // responseEnd - startTime
             const spReqDuration = response.headers.get("sprequestduration");
 
-            const resources1 = performance.getEntriesByType("resource");
+            // getEntriesByType is only available in browser performance object
+            const resources1 = performance.getEntriesByType?.("resource") ?? [];
```

Comment:
@raandra - this change should be mostly nonfunctional for browsers.
For nodejs, I believe there isn't a direct equivalent of this `performance.getEntriesByType`, so this change should just leave them at their default -1 values by providing an empty array to `resources1`.  Is this reasonable behavior for the logs?

Reply:
Do you mean that when I run npm run start:spo-df and test on localhost:8081/<app>, I won't see the data? Or something else? @ram-ramasubramanian FYI, keeping you in loop.

Reply:
"Nonfunctional" as in no change to the existing behavior, as opposed to making the code do nothing.  (But it will still do nothing running under Node)

Reply:
Then, I'm fine with the proposed change.

Reply:
Oh yes, thanks @heliocliu for clarifying. I meant that for browsers this should not change behavior.
So with `npm run start:spo-df`, everything should still work since that's running in the browser.

If we had a client running on Node.js, they would be missing out on this telemetry, and those properties would remain at their default -1 values. I was asking if that part is reasonable behavior?